code/go/0chain.net/validator/main.go

```diff
@@ -171,7 +171,7 @@ func RegisterValidator() {
 			time.Sleep(transaction.SLEEP_FOR_TXN_CONFIRMATION * time.Second)
 			t, err := transaction.VerifyTransaction(txnHash, chain.GetServerChain())
 			if err == nil {
-				txnVerified = true
+				txnVerified = true //nolint:ineffassign // probably has something to do with goroutines
```

Comment:
This line can saftly be removed as you are just about to return. It has some documentary value, but remove it.